doc/user/external.xml

```diff
@@ -283,4 +285,24 @@ env2.CompilationDatabase('compile_commands-linux64.json')
     </programlisting>
 
   </section>
+
+    <section>
+        <title>Ninja Build Generator</title>
+
+        <para>
+            This is an experimental new feature.
+        </para>
+        <para>
+            Using the
```

Comment:
placeholder content? Seems odd to have three such short paragraphs...

Reply:
I've filled this in a bit in the next push.